build_docs.py

```diff
@@ -1147,6 +1151,21 @@ def build_docs(args) -> bool:
     return all_built_successfully
 
 
+@cache
```

Comment:
This doesn't hurt, but are we likely to format the same (float) number of seconds more than once?

Reply:
Yeah, I'll remove it. I initially thought it was seconds, but even then it won't make much difference here.